content/docs/ui/sending-email/deliverability.md

```diff
@@ -73,10 +73,10 @@ With email, things don't happen overnight, and magic wands are few and far betwe
 
 Encouraging your recipients to do certain things can help bolster the trust ISPs have for you and your messages. Some examples can include:
 
-- "Add us to your address book!" - Having a recipient add your from address you their address book or trusted senders list can go a long way. More often than not, if one of an ISPs recipients trust a sender, they will be more lenient to similar messages to different recipients!
+- "Add us to your address book!" - Having a recipient add you from their address book or trusted senders list can go a long way. More often than not, if one of an ISPs recipients trust a sender, they will be more lenient to similar messages to different recipients!
```

Comment:
Should read "Having a recipient add your *from* address to their address book or trusted senders list can go a long way."